backend/src/apiserver/server/pipeline_server.go

```diff
@@ -128,6 +128,23 @@ func (s *PipelineServer) CreatePipeline(ctx context.Context, request *api.Create
 	return ToApiPipeline(pipeline), nil
 }
 
+func (s *PipelineServer) UpdatePipelineDefaultVersion(ctx context.Context, request *api.UpdatePipelineDefaultVersionRequest) (*empty.Empty, error) {
+	if s.options.CollectMetrics {
+		getPipelineRequests.Inc()
+	}
+
+	err := s.resourceManager.UpdatePipelineDefaultVersion(request.PipelineId, request.VersionId)
+	if err != nil {
+		return nil, util.Wrap(err, "Delete pipelines failed.")
+	}
+	if s.options.CollectMetrics {
+		pipelineCount.Dec()
+	}
```

Comment:
nit: did you copy this from a different method?
I think you should remove them

Reply:
Yeah I did copy it :). Ok, will do so.